### ISSUE TITLE
Weight decay per group from config

### ISSUE THREAD

**Issue by VladVin** [eora-ai/torchok#83]
**Is your feature request related to a problem? Please describe.**
A user can specify model layers with their own weight decay parameters being set in the training configuration.

**Describe the solution you'd like**
In the config file it might be a dict like this:
```yaml
weight_decays:
  'head.layer1': 0.00001
  'head.layer2': 0.00005
```

**Describe alternatives you've considered**

**Additional context**

**Comment by Animatory** [comment]
Implemented in https://github.com/eora-ai/torchok/pull/152/files
